process/transactionProcessor.go

```diff
@@ -87,13 +90,22 @@ func NewTransactionProcessor(
 	if newTxCostProcessor == nil {
 		return nil, ErrNilNewTxCostHandlerFunc
 	}
+	jsonMarshalizer, err := marshalFactory.NewMarshalizer("json")
```

Comment:
why use factory when you could directly init with the json marshaller?

Reply:
done.